ebean-test/src/test/java/org/tests/json/TestJsonMapBasic.java

```diff
@@ -70,7 +70,7 @@ public void whereManyPredicateDb2() {
     bean.getDetails().add(new EBasicJsonMapDetail("db2-detail2"));
 
     DB.save(bean);
-    Query<EBasicJsonMap> query1 = DB.find(EBasicJsonMap.class).select("*").fetch("details").where()
+    Query<EBasicJsonMap> query1 = DB.find(EBasicJsonMap.class).select("id, name, version").fetch("details").where()
```

Comment:
Hier wirft uns DB2 einen SQLError, weil vorher versucht wurde einen SELECT DISTINCT auf t0.content gemacht wird. content ist aber ein clob und es kommt zum Error.

Reply:
Ja, dann war ich vllt. hier auf der falschen Spur. Aber auf jedenfall laufen wir in den SQLCODE Error: -134
https://www.ibm.com/docs/en/db2-for-zos/12?topic=codes-134